app/javascript/app/components/nav/nav-component.jsx

```diff
@@ -2,24 +2,50 @@ import React, { PureComponent } from 'react';
 import PropTypes from 'prop-types';
 import Link, { NavLink } from 'redux-first-router-link';
 import cx from 'classnames';
-import { Icon } from 'cw-components';
+import environmentalAffairsLogo from 'assets/environmental-affairs-logo.png';
+import SAflag from 'assets/south-africa-flag.png';
 
-import cwLogo from 'assets/cw-logo.svg';
 import styles from './nav-styles.scss';
 
 class Nav extends PureComponent {
   render() {
     const { routes, className } = this.props;
     return (
-      <nav className={cx(styles.navbar, className)}>
-        <Link className={styles.link} to="/">
-          <Icon theme={{ icon: styles.logo }} icon={cwLogo} />
-        </Link>
-        {routes.map(route => (
-          <NavLink className={styles.link} key={route.label} to={route.link} activeClassName={styles.active}>
-            {route.label}
-          </NavLink>
-        ))}
+      <nav className={cx(styles.header, className)}>
+        <div className={styles.navbarContainer}>
+          <div className={styles.navElement}>
+            <Link className={styles.link} to="/">
+              <img
+                src={environmentalAffairsLogo}
+                alt="Environmental Affairs Logo"
+                className={cx(styles.logo)}
+              />
+            </Link>
+          </div>
+          <div className={cx(styles.navElement, styles.pageTitleContainer)}>
+            <span className={styles.climatText}>Climate</span>
```

Comment:
IMO this could be an `<h1>` tag as it is the main title of the page

Reply:
Ah! and this is the link going for the root page

Reply:
👍